SetupContextMenu.ps1

```diff
@@ -78,14 +78,17 @@ Write-Host "Add top layer menu (background) => $contextMenuRegPath"
 
 # Get Windows terminal profile
 $rawContent = (Get-Content $wtProfilesPath) -replace '^\s*\/\/.*' | Out-String
-$profiles = (ConvertFrom-Json -InputObject $rawContent).profiles.list
+$profiles = (ConvertFrom-Json -InputObject $rawContent).profiles
```

Comment:
You may use the old style profiles.
Could you back up your current profiles and generate new one?

Reply:
Oh it's true, you're right but its weird, the file don't update when the terminal update, Sorry for the change